index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <link rel="stylesheet" href="style.css">
+  <script src="https://kit.fontawesome.com/ad79131c56.js" crossorigin="anonymous"></script>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+<body>
+  <header class="nav1">
+    <a class="logo"  href="#">My logo</a>
+      <div class="nav2">
+        <img class="union" src="img/Union.svg" alt="">
+          <ul class="first-list">
+            <li>
+              <a href="#Portfolio">Portfolio</a>
+            </li>
+            <li>
+              <a href="#About">About</a>
+            </li>
+            <li>
+              <a href="#Contact">Contact</a>
+          </li>
+         </ul>
+      </div>
+    </header>
+    <section id="section1">
+    <div class="text1">
+      <h1>I'm Stanley Enekwe, Glad to see you!</h1>
+      <p class="header-text">I’m a software developer! I can help you build a product , 
+        feature or website Look through some of my work and experience! 
+        If you like what you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+    </div> 
+    <div class="connect">
+        <a href="#">LET'S CONNECT</a>
+    </div>
+    <div class="icon1">
+      <ul>
+          <li><a href="#"><i class="icons fab fa-twitter" aria-hidden="true"></i></a></li>
+          <li><a href=""><i class="icons fab fa-linkedin-in" aria-hidden="true"></i></a></li>
+          <li><a href="#"><i class="icons fab fa-medium-m" aria-hidden="true"></i></a></li>
+          <li><a href=""><i class="icons fab fa-github" aria-hidden="true"></i></a></li>            <li><a href="#"><i class="icons fab fa-angellist" aria-hidden="true"></i></a></li>
+      </ul>
```

Comment:
- [ ] OPTIONAL: Please consider decreasing the size of buttons and changing color to the provided one which is `#505F79`: